index.html

```diff
@@ -70,6 +83,8 @@ <h2>Moon</h2>
                 <a href="#">Contact</a>
             </nav>
```

Comment:
You could add about into this footer nav bar now... the page exists. Remember to link it correctly.